specification/applicationinsights/data-plane/microsoft.insights/preview/2018-04-20/swagger.json

```diff
@@ -251,11 +251,6 @@
             "$ref": "#/parameters/apiVersion"
           }
         ],
-        "produces": [
```

Comment:
The spec declares "produces": "application/json" globally and actually sending something like "Accept" "application/json;odata=none" will not work with the backend.